build/debian-hyperkube-base/Dockerfile

```diff
@@ -35,6 +35,8 @@ RUN echo CACHEBUST>/dev/null && clean-install \
     openssh-client \
     nfs-common \
     socat \
-    util-linux
+    util-linux && \
+    echo "deb http://ftp.debian.org/debian jessie-backports main contrib" > /etc/apt/sources.list.d/jesse-backports.list \
```

Comment:
Rather than add jessie-backports, perhaps we move up to using stretch? @ixdy @tallclair